tests/pkg_cmsis-dsp/tests/01-run.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+def testfunc(child):
+    child.expect(r"dsp:   0\.903941\d{3}")
```

Comment:
I only used six 6 significant numbers here, because the output states "*roughly* 7 decimals [sic] precision"